### PR TITLE
Updates to Design and Develop APIs 

### PR DESCRIPTION
**Pull request by juliatetrud**

### CODE REVIEW

File: v6/postman/design_and_develop_apis/introduction_to_apis.md [hunk 1]
```diff
@@ -6,39 +6,33 @@ warning: false
...
+For example, a test engineer working on test suites connected to an API might be collaborating with a technical writer and a developer. The writer would be focused on API documentation, while the developer is working on mocks and monitors attached to the API. Now, all three - test engineer, tech writer, and developer - can on the same API definition in Postman, rather than having to work within the same Postman collection. The API itself is the collaboration point, and collections tagged to the API can represent different use cases for the API.
 
-With APIs in Postman, you can perform the following tasks:
+* Create and manage APIs in your workspace, keep them consistent and backward-compatible
```
Comment (tristandenyer) [inline v6/postman/design_and_develop_apis/introduction_to_apis.md]: Shouldn't this be "Design and Develop APIs..."?

File: v6/postman/design_and_develop_apis/managing_apis.md [hunk 2]
```diff
@@ -13,71 +13,72 @@ APIs reside in your workspace, just like your collections. This section walks yo
...
-Navigate to the APIs tab and click **Create an API** button, shown in a red circle in the above screen. The following screen opens.
+Navigate to the APIs tab and click **Create an API** button, shown in a red circle in the above screen. The following screen opens.  
 
 [![api create4](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Create4-Dashboard.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Create3-Dashboard.png)
 
-Specify a name for your API and click the '&#9745;' adjacent to the name. Your API is created.
+Specify a name for your API and click the '&#9745;' adjacent to the name. Your API is created. 
 
-You can create different API versions from a specific version of your API. For more information on this, refer to the section [Creating API versions](/docs/v6/postman/design_and_develop_apis/versioning-an-api).
+You can create different API versions from a specific version of your API. For more information on this, refer to the section [Creating API versions](/docs/v6/postman/working_with_apis/versioning-an-api).
```
Comment (tristandenyer) [inline v6/postman/design_and_develop_apis/managing_apis.md]: This URL should be: /docs/postman/design_and_develop_apis/versioning_an_api/

File: v6/postman/design_and_develop_apis/managing_apis.md [hunk 3]
```diff
@@ -13,71 +13,72 @@ APIs reside in your workspace, just like your collections. This section walks yo
...
+Click the ellipsis (...) next to the API, and select **Delete API**, as indicated by the red arrow in the following screen:
 
 [![api delete](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Delete1.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Delete1.png)
 
 A delete confirmation message appears:
 
 [![api delete2](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Delete2.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Delete2.png)
 
-When you delete an API, the collections, monitors, mocks, and environments linked to it are not deleted. For a more detailed understanding on this topic, refer to [Sharing APIs](/docs/v6/postman/design_and_develop_apis/sharing_apis)
+When you delete an API, the collections, monitors, mocks, and environments linked to it are not deleted. For a more detailed understanding on this topic, refer to [Sharing APIs](/docs/v6/postman/working_with_apis/sharing_apis)
```
Comment (tristandenyer) [inline v6/postman/design_and_develop_apis/managing_apis.md]: This URL should be /docs/postman/design_and_develop_apis/sharing_apis/

File: v6/postman/design_and_develop_apis/managing_apis.md [hunk 4]
```diff
@@ -13,71 +13,72 @@ APIs reside in your workspace, just like your collections. This section walks yo
...
 
-You can also delete specific versions of your APIs. For more information on this, refer to the section [Deleting an API version](/docs/v6/postman/design_and_develop_apis/versioning-an-api).
+You can also delete specific versions of your APIs. For more information on this, refer to the section [Deleting an API version](/docs/v6/postman/working_with_apis/versioning-an-api).
```
Comment (tristandenyer) [inline v6/postman/design_and_develop_apis/managing_apis.md]: This URL should be: /docs/postman/design_and_develop_apis/versioning_an_api/

File: v6/postman/design_and_develop_apis/managing_apis.md [hunk 5]
```diff
@@ -13,71 +13,72 @@ APIs reside in your workspace, just like your collections. This section walks yo
...
 
 ### Removing an API from a workspace
 
-From the Postman app, you can remove an API from the sidebar. Click the ellipsis (...) next to the API you would like to remove, select **Remove from workspace** and confirm that you would like to remove the API from the current workspace. The API will no longer be visible in the workspace.
+From the Postman app, you can remove an API from the sidebar. Click the ellipsis (...) next to the API you would like to remove, select **Remove from workspace** and confirm that you would like to remove the API from the current workspace. The API will no longer be visible in the workspace. 
 
-When you remove an API from a workspace, the collections, monitors, mocks, and environments linked to it are not removed. For a more detailed understanding on this topic, refer to [Sharing APIs](/docs/v6/postman/design_and_develop_apis/sharing_apis).
+When you remove an API from a workspace, the collections, monitors, mocks, and environments linked to it are not removed. For a more detailed understanding on this topic, refer to [Sharing APIs](/docs/v6/postman/working_with_apis/sharing_apis).
```
Comment (tristandenyer) [inline v6/postman/design_and_develop_apis/managing_apis.md]: This URL should be /docs/postman/design_and_develop_apis/sharing_apis/

File: v6/postman/design_and_develop_apis/managing_apis.md (hunk 4 above)
Comment (juliatetrud) [reply v6/postman/design_and_develop_apis/managing_apis.md]: Yeah. This is strange. I thought I had fixed all of these URLs. Good catch.

**Pull request merged**